src/components/Home/Main/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import "../../../index"
+import { DealsCard } from "../common/deals-card"
+
+export function Main() {
+    return (
+        <main className="main-container">
+            {/* main banner */}
+            <section className="main-container top-section">
```

Comment:
If can then convert the banner into a different component.

Reply:
Done